scripts/sanity-check.sh

```diff
@@ -21,22 +21,12 @@ fi
 results=$(./scripts/get-from-collector.sh "$ENDPOINT" "$COLLECTOR_USERNAME" "$COLLECTOR_PASSWORD")
 start_index=${results%%[*} start_index=$((${#start_index} + 1))
 results="${results:start_index-1}"
-results_test_ids=$(echo "$results" | jq -r '.[-1].ClaimResults[].test_id')
+results_claim=$(echo "$results" | jq -r '.[-1].Claim')
 
-# Get generated policy requiredPassingTests ids
-GENERATED_POLICY_RAW_URL="https://raw.githubusercontent.com/test-network-function/cnf-certification-test/main/generated_policy.json"
-read -d '' -ra required_test_ids <<<"$(curl $GENERATED_POLICY_RAW_URL | jq -r '.grades.requiredPassingTests[].id')"
-
-# Ensure all ids in ids_array are in results from collector
-echo "Iterating over required passing tests ids..."
-for test_id in "${required_test_ids[@]}"; do
-	if [[ "${results_test_ids[*]}" == *"$test_id"* ]]; then
-		echo "test $test_id exists in the collector"
-	else
-		echo "test $test_id does not exist in the collector"
-		echo "Collector's sanity check has failed."
-		exit 1
-	fi
-done
+if [ -z "$results_claim" ]; then
+	echo "User ${COLLECTOR_USERNAME} doesn't have any claims stored."
+	echo "Collector's sanity check has failed!."
```

Comment:
There is no need for the point after the exclamation mark. Also, having a single error message is a good approach.